src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -73,4 +73,11 @@
 "Description"="Disables the Razor Language Server so that only the Cohost server is active. This is probably a bad idea to turn on if you are not on the Razor Tooling team."
 "Value"=dword:00000000
 "Title"="Disable Razor Language Server (requires restart)"
-"PreviewPaneChannels"="IntPreview,int.main"
\ No newline at end of file
+"PreviewPaneChannels"="IntPreview,int.main"
+
+[$RootKey$\FeatureFlags\Razor\LSP\EnableFuse]
+"Description"="Enable combined design time/runtime code generation for Razor files"
+"Value"=dword:00000000
+"Title"="Enable 'fuse' for Razor files (requires restart)"
+"PreviewPaneChannels"="*"
```

Comment:
Doesn't this mean we show it to all users? We probably just want `IntPreview,int.main` like the above (especially if we're sticking with "fuse" 😛)

Reply:
I always thought `VisibleToInternalUsersOnlyChannels` would make it so only internal users can see the feature flag, regardless of build. I'm happy to copy a different random line than the one I did though

Reply:
I have NO idea :)